low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -469,7 +469,7 @@ def get_realstor_encl_data(self, fru: str):
             "sensors": self.URI_CLIAPI_SHOWSENSORSTATUS,
             "volumes": self.URI_CLIAPI_SHOWVOLUMES,
             "disk-groups": self.URI_CLIAPI_SHOWDISKGROUPS,
-            "enclosures": self.URI_CLIAPI_SHOWENCLOSURE,
+            "storage": self.URI_CLIAPI_SHOWENCLOSURE,
```

Comment:
Here, All calls are enclosure specific/ do we need to change here too?
cc @cdeshmukh

Reply:
I raised the same and the answer was both storage and enclosure are meaning to one (storage enclosure).